### PR TITLE
Skill - Boon

### PR DESCRIPTION
**Pull request by JesterWizard**
Heal bad status effects at the beginning of the turn.

SIDE NOTE:
I couldn't get the `StartStatusHealEffect` animation to play for some reason. But the skill works otherwise.

### CODE REVIEW

File: Contants/Texts/Source/texts/Skills_PrePhase.txt [hunk 1]
```diff
@@ -49,4 +49,9 @@ who's RES is 3 less than unit's[X]
...
 RainDish:[NL]
 If it is raining, then restore 30%[NL]
-of max HP at the start of each turn.[X]
\ No newline at end of file
+of max HP at the start of each turn.[X]
+
+## MSG_SKILL_Boon
+Boon:[NL]
+Heal bad status effects at the[NL]
+beginning of the turn.[X]
```
Comment (MokhaLeee) [inline Contants/Texts/Source/texts/Skills_PrePhase.txt:57]: add a blank line at the end of file

File: include/constants/skills-others.enum.txt [hunk 2]
```diff
@@ -125,3 +125,4 @@ SID_FairyTaleFolk
 
 // Pre-phase skills
 SID_RainDish
+SID_Boon
```
Comment (MokhaLeee) [inline include/constants/skills-others.enum.txt:128]: the same

Comment (JesterWizard) [reply include/constants/skills-others.enum.txt]: Done

File: Contants/Texts/Source/texts/Skills_PrePhase.txt (hunk 1 above)
Comment (JesterWizard) [reply Contants/Texts/Source/texts/Skills_PrePhase.txt]: Done

**Comment by MokhaLeee** [comment]
Yes, this PR does have both functionality and performance issues and need fix

**Pull request merged**